### PR TITLE
Feature/filter data update

### PR DESCRIPTION
**Pull request by noahmanger**
Various changes to stay in sync with https://github.com/18F/fec-cms/pull/256 and https://github.com/18F/fec-style/pull/277
- Re-arranges filters and tags so that the tag bar is on top of the filter panel
- Creates a new datatable.html template and routes all datatable URLs to it, simplifying the steps to make a new datatable page

(Note: I pulled in the release to this branch, but not sure if we want to merge that in separately before this).

### CODE REVIEW

**Comment by noahmanger** [comment]
Hmmm. Seeing a new error here: 
```
PhantomJS 2.1.1 (Linux 0.0.0) ERROR
You need to include some adapter that implements __karma__.start method!
```

@ccostino or @jmcarp any idea what that's about?

**Comment by ccostino** [comment]
@noahmanger, I'm seeing an error before that:

```
25 03 2016 14:02:34.896:ERROR [framework.browserify]: bundle error
25 03 2016 14:02:34.900:ERROR [framework.browserify]: 
/home/travis/build/18F/openFEC-web-app/node_modules/fec-style/js/templates/election-cycles.hbs:1
{{#each this}}
  ^
ParseError: Unexpected character '#'
```

This doesn't appear to be a problem introduced in the last few commits that failed, but just to be sure, is that the correct syntax?

**Comment by noahmanger** [comment]
Aaaaah. You know what, I _did_ [update the version of Handlebars](https://github.com/18F/openFEC-web-app/pull/1140/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2L71) to use the same version as fec-style and fec-cms in order to resolve the issue we were running into where templates were compiled with different versions. 

I wonder if that's related. Looking at release notes to try to figure it out.

**Comment by noahmanger** [comment]
Ok, so I tried reverting back to v3.0.3 and still have the same error. And I'm not seeing anything in the [release notes](https://github.com/wycats/handlebars.js/blob/master/release-notes.md). So...

**Comment by noahmanger** [comment]
I'm still stumped on this. It seems to be getting caught up on the simple requiring of a `.hbs` template. I can't figure out what I would have changed to have affected this, and things run smoothly on develop. @jmcarp if you have a minute can you take a look at why tests are failing?

**Comment by noahmanger** [comment]
Ok! Tests are passing now. Ready for review / merge.

File: package.json [hunk 1]
```diff
@@ -60,15 +60,15 @@
...
+    "glossary-panel": "0.1.2",
     "chroma-js": "1.0.1",
     "colorbrewer": "0.0.2",
     "component-sticky": "1.0.0",
     "d3": "3.5.5",
     "datatables.net": "1.10.10",
     "datatables.net-responsive": "2.0.1",
-    "fec-style": "1.7.0",
-    "handlebars": "3.0.3",
+    "fec-style": "git+https://github.com/18f/fec-style#filter-data-update",
```
Comment (ccostino) [inline package.json]: Just don't forget to change this back when we're ready to merge!

File: openfecwebapp/templates/datatable.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+{% include 'partials/datatable-modal.html' %}
+
+{% endblock %}
+
+{% block scripts %}
+{% if office %}
+<script>
+var context = {
+  office: '{{ office }}'
+};
```
Comment (ccostino) [inline openfecwebapp/templates/datatable.html]: What is this `context` and `office` variable/property for, and why is it defined here?

File: package.json [hunk 3]
```diff
@@ -67,8 +67,8 @@
     "d3": "3.5.5",
     "datatables.net": "1.10.10",
     "datatables.net-responsive": "2.0.1",
-    "fec-style": "~1.7",
-    "handlebars": "3.0.3",
+    "fec-style": "git+https://github.com/18f/fec-style#filter-data-update",
```
Comment (ccostino) [inline package.json]: Just need to update to point to next minor version, should be `~1.8` once we're ready with `fec-style`.

File: openfecwebapp/templates/datatable.html (hunk 2 above)
Comment (ccostino) [inline openfecwebapp/templates/datatable.html]: Just adding a note that @noahmanger and I reviewed this together and all is fine for now.

**Comment by noahmanger** [comment]
Travis is still failing but it's giving me an error when I try to clear the cache (on both the app and CLI). I'm going to close out these pull requests and file fresh ones to see if that does it.

**Pull request closed without merging**